src/settings-ui/Settings.UI/Views/PowerLauncherPage.xaml

```diff
@@ -109,6 +109,28 @@
                             <CheckBox x:Uid="PowerLauncher_ClearInputOnLaunch" IsChecked="{x:Bind Mode=TwoWay, Path=ViewModel.ClearInputOnLaunch}" Margin="{StaticResource ExpanderSettingMargin}" />
                         </controls:SettingExpander.Content>
                     </controls:SettingExpander>
+                    <controls:SettingExpander IsExpanded="True">
```

Comment:
Let us switch this with the Expander before as you search first before you see the results.